Source/JavaScriptCore/jit/OperationResult.h

```diff
@@ -90,7 +90,6 @@ template<typename T>
 using OperationReturnType = std::conditional_t<(std::is_same_v<T, void> || canMakeExceptionOperationResult<T>), ExceptionOperationResult<T>, T>;
 
 // This class exists so we can cast ExceptionOperationImplicitResult<From> to ExceptionOperationResult<To> implicitly while ExceptionOperationResult remains POD.
-// FIXME: we could use __atribute__((trivial_abi)) when we no longer support MSVC.
```

Comment:
I think Ross meant to use the attribute and remove the class rather than delete the comment.

Reply:
Yeah, since that involves removing this implicit coercion class, I'm reverting the removal and the typo fix to minimize this PR.